r2dbc-spi/src/main/java/io/r2dbc/spi/UserPasswordCredentialImpl.java

```diff
@@ -0,0 +1,23 @@
+package io.r2dbc.spi;
+
+public class UserPasswordCredentialImpl implements UserPasswordCredential {
```

Comment:
I think this class can be a package-private (ie: no "public" modifier). Code that is external to the package should use the factory method.

Reply:
Agreed, resolved with commit 2b09f64